### DISCUSSION TITLE
Faces with face mask not recognized

### DISCUSSION THREAD

**Discussion by deepfriedbrain** [photoprism/photoprism#3799] [Q&A]
I just installed Photoprism inside a Docker container on Raspberry Pi 4. 

I've added a few photos and noticed that it's not able to recognize faces where people have face masks on. Is there a configuration that can enable this. I tried searching for an answer but couldn't find a solution to this problem.

**Comment by alternativesurfer** [comment]
I am interested in this as well.

Any idea what [face recognition](https://docs.photoprism.app/getting-started/config-options/#face-recognition) options can be tweaked to allow for masked faces to be detected?

Thanks!

**Reply by lastzero** [reply]
Probably none as the model hasn't been trained with masked faces.

**Reply by alternativesurfer** [reply]
Do you think it would be possible for PhotoPrism to implement something like DigiKam uses? Where you can train the model itself?

https://docs.digikam.org/en/main_window/people_view.html
